src/main/java/com/jalasoft/selenium/bruno/movies/Customer.java

```diff
@@ -0,0 +1,95 @@
+package com.jalasoft.selenium.bruno.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.jalasoft.selenium.bruno.movies.NewRelease.NEW_RELEASE;
+
+/**
+ * This class represents a Customer.
+ */
+class Customer {
+    private static final String BRAKE_LINE = "\n";
+    private static final String TABULATION = "\t";
+    private String customerName;
+    private List<Rental> rentalList = new ArrayList<>();
+
+    /**
+     * Constructor of a Custumer.
+     *
+     * @param name of customer.
+     */
+    Customer(final String name) {
+        customerName = name;
+    }
+
+    /**
+     * This method adds a new rental.
+     *
+     * @param newRental to be added.
+     */
+    public void addRental(final Rental newRental) {
+        rentalList.add(newRental);
+    }
+
+    /**
+     * This method gets the customer name.
+     *
+     * @return customer name.
+     */
+    public String getName() {
+        return customerName;
+    }
+
+    /**
+     * This method creates the ticket.
+     *
+     * @return the ticket
+     */
+    public String statement() {
```

Comment:
this method is doing 3 things. it is calculating the totalAmount, totalFrequentRenterPoints and printing.
Please split it.